src/pages/About.vue

```diff
@@ -29,6 +146,38 @@ export default {
   },
   components: {
     History
+  },
+  data() {
+    return {
+      svc_img: "/assets/img/cloud.jpg"
+    };
+  },
+  methods: {
+    mouseOverChangeImage(i_clicked, newSrc) {
+      //console.log(elem_clicked.target)
+      var all_cards = document.getElementsByClassName(
+        "button card box is-white is-medium"
+      );
+      var active_card = document.getElementsByClassName(
+        "button card box is-white is-medium active-card"
+      );
+      for (var i = 0; i < active_card.length; i++) {
+        active_card[i].classList.remove("active-card");
+      }
+      all_cards[i_clicked].classList.add("active-card");
+      this.svc_img = newSrc;
```

Comment:
In Vue it would probably be easier to manage the DOM state by using JS in the template rather than manipulating the document by hand. Check out this snippet below:

```
<template>
... Example of optional class
  <div class="button box" :class="{ selected_image == 1: 'active-card' }"></div>
...
</template>
  
<script>
  data() {
    return {
      // This way we can record the state of what image we have selected
      selected_image: 0,
      images: [
        "/assets/img/cloud.jpg",
        "/assets/img/cloud2.jpg",
        "/assets/img/cloud3.jpg"
      ]
    };
  },
  methods: {
    mouseOverChangeImage(i_clicked, newSrc) {
      this.selected_image = i_clicked
...
</script>
```